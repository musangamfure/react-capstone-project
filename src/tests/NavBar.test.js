import { render } from '@testing-library/react';
import Header from '../components/NavBar';

test('renders NavBar component', () => {
  const { container } = render(<Header />);
  expect(container.innerHTML).toMatchSnapshot();
});
