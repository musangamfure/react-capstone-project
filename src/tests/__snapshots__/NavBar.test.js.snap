// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders NavBar component 1`] = `"<header class=\\"header\\"><h1>Countries App</h1><div class=\\"theme-dark\\" role=\\"button\\" tabindex=\\"0\\"><i class=\\"fa-solid fa-sun\\"></i></div></header>"`;
